internal/backends/rlang/rlang.go

```diff
@@ -12,38 +14,72 @@ func getImports(imports string) []string {
 	return regexp.MustCompile(`[a-zA-Z_]\w*`).FindAllString(imports, -1)
 }
 
+func getRPkgDir() string {
+	if rLibsUser := os.Getenv("R_LIBS_USER"); rLibsUser != "" {
+		return rLibsUser
+	}
+
+	return path.Join(os.Getenv("HOME"), "R", "x86_64-pc-linux-gnu-library", "3.4")
+}
+
 func createRPkgDir() {
-	if _, err := os.Stat("./R"); os.IsNotExist(err) {
-		if err = os.MkdirAll("./R/x86_64-pc-linux-gnu-library/3.4", os.ModeDir); err != nil {
+	dir := getRPkgDir()
+
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
+		if err = os.MkdirAll(dir, os.ModeDir+os.ModePerm); err != nil {
 			panic(err)
 		}
 	} else if err != nil {
 		panic(err)
 	}
 }
 
-func updateLibPaths() {
+func installRPkg(name string) {
+	if strings.Contains(name, "'") {
+		name = strings.ReplaceAll(name, "'", `\'`)
+	}
+
+	// Checking this first makes sure that we aren't running regex on everything
+	if strings.HasSuffix(name, `\`) {
+		suffix := regexp.MustCompile(`\\+$`).FindString(name)
+		if len(suffix)%2 == 1 {
+			name += `\`
+		}
+	}
+
 	util.RunCmd([]string{
 		"R",
```

Comment:
How does R know here to install the package?

Reply:
Because that's where it installs to by default if it doesn't have write access to its normal library directory on linux (and it usually doesn't). For non-linux platforms (and well, it works on linux too), R can use the `$R_LIBS_USER` env var instead

Reply:
Got it.